sample-build-scripts/test/xamarin-uitest-android/appcenter-post-build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
```

Comment:
I would add `set -e` here at the top to make sure that the script bails out if there are any statements that return an error.